code/_onclick/xeno.dm

```diff
@@ -21,6 +21,16 @@
 
 	A.attack_larva(src)
 
-
 /atom/proc/attack_larva(mob/living/carbon/xenomorph/larva/L)
 	return
+
+
+
+/mob/living/carbon/xenomorph/hivemind/UnarmedAttack(atom/A, proximity_flag)
+	if(lying)
```

Comment:
Reminder for myself to change this lying to `handblock_flags` in the mobility refactor.